clean_newsarticles.py

```diff
@@ -0,0 +1,73 @@
+
+import re
+import csv
+
+#input_file = r'data\novinove_clanky\Fake News Detection\fake.csv'
+
+#pattern pro webove stranky https a twitter musi jit out
+
+
+def read_csv_to_dict(input_file):
+    """Reads a CSV file and returns the data as a list."""
+    content_list = [] 
+    try: 
+        with open(input_file, 'r', encoding='utf-8') as f:
+            reader = csv.reader(f) 
+            for raw in reader:  
+                    content_list.append(raw)
+        del content_list[-3:]
+        return content_list
+   
+    
+    except UnicodeDecodeError as e:
+        print(f"Encoding error: {e}")
+    # Try alternative encodings if UTF-8 fails
+    try:
+        with open(input_file, 'r', encoding='latin-1') as f:
+            reader = csv.reader(f)
+            for raw in reader:  
+                    content_list.append(raw) 
+        return content_list
+    except Exception as alt_e:
+        print(f"Alternative encoding failed: {alt_e}")
+
+
+def remove_string(content_list, pattern_list):
+    cleaned_list = []
+    for raw in content_list:
+        if isinstance(raw, list):
+            cleaned_row = []
+        for text in raw:

```

Comment:
It’s better to use a tab or indentation for the loop 'for text in raw:' to ensure no error occurs if the content of `raw` is `None`